docs/agentic-development/roadmap/openshell/sandbox-bun-crash-report.md

```
@@ -0,0 +1,90 @@
+   ```
+
+2. Connect and run Claude:
+   ```bash
+   openshell sandbox connect my-project
+   cd /sandbox
+   claude
+   ```
+
+## Observed Behavior
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)